lib/runtime/wasmer/exports.go

```diff
@@ -297,13 +306,17 @@ func (in *Instance) GrandpaGenerateKeyOwnershipProof(authSetID uint64, authority
 		return nil, err
 	}
 
-	keyOwnershipProof := types.GrandpaOpaqueKeyOwnershipProof{}
+	var keyOwnershipProof *types.GrandpaOpaqueKeyOwnershipProof
 	err = scale.Unmarshal(encodedOpaqueKeyOwnershipProof, &keyOwnershipProof)
```

Comment:
```suggestion
	err = scale.Unmarshal(encodedOpaqueKeyOwnershipProof, keyOwnershipProof)
```

Reply:
Yes that is the intended behavior now. Scale takes a pointer to the object, which in this case is a pointer, so a pointer to a pointer is intentional.